syntax = "proto3";

package Protocol;
message LoginRequest {
  string email = 1;
  string password= 2;
}